paddle/fluid/inference/api/paddle_analysis_config.h

```diff
@@ -145,7 +145,7 @@ struct PD_INFER_DECL AnalysisConfig {
   ///
   /// \param[in] other another AnalysisConfig
   ///
-  explicit AnalysisConfig(const AnalysisConfig& other);
+  AnalysisConfig(const AnalysisConfig& other);
```

Comment:
Hi why do you remove the explicit keyword? Maybe do not change Baidu original API

Reply:
I removed it because `explicit` cause problems with copy elision.
Since it's kinda hard to find any recommendation to create such constructors, I think it was added by mistake.
I hope that someone from PaddlePaddle team will review it.

Reply:
@jiangjiajun Please review